eth/tracers/native/prestate.go

```diff
@@ -158,6 +160,9 @@ func (t *prestateTracer) OnTxStart(env *tracing.VMContext, tx *types.Transaction
 	t.lookupAccount(from)
 	t.lookupAccount(t.to)
 	t.lookupAccount(env.Coinbase)
```

Comment:
When this is added here, should we add accounts which receive fees here as well, like the `FeeHandler`?

Reply:
I only thought about the fee currency txs in this PR, where the `FeeHandler` is part of the fee currency's storage. But for a normal tx and enables Cel2, that might be necessary. I'll take a look at that case.

Reply:
Followup issue is at https://github.com/celo-org/op-geth/issues/216.